Makefile

```diff
@@ -89,7 +87,7 @@ run: $(ISO)
 debug: ## build and run the OS in debug mode
 debug: CFLAGS += $(DEBUG_CFLAGS)
 debug: $(ISO)
-	$(QEMU) -cdrom $< -serial file:/tmp/serial.log
+	$(QEMU) -cdrom $< -serial file:/tmp/serial.log -m 1G
```

Comment:
Set the amount of RAM for 1GB since it could cause crashes otherwise.

Reply:
That's interesting. I suspect a problem with the memory mapping but I am not sure.